internal/services/oracle/oracle_autonomous_database_cross_region_disaster_recovery_resource_test.go

```diff
@@ -0,0 +1,286 @@
+  license_model                    = azurerm_oracle_autonomous_database.adbs_primary_for_crdr.license_model
+  backup_retention_period_in_days  = azurerm_oracle_autonomous_database.adbs_primary_for_crdr.backup_retention_period_in_days
+  auto_scaling_enabled             = azurerm_oracle_autonomous_database.adbs_primary_for_crdr.auto_scaling_enabled
+  auto_scaling_for_storage_enabled = azurerm_oracle_autonomous_database.adbs_primary_for_crdr.auto_scaling_for_storage_enabled
+  mtls_connection_required         = azurerm_oracle_autonomous_database.adbs_primary_for_crdr.mtls_connection_required
+  data_storage_size_in_tbs         = azurerm_oracle_autonomous_database.adbs_primary_for_crdr.data_storage_size_in_tbs
+  compute_model                    = azurerm_oracle_autonomous_database.adbs_primary_for_crdr.compute_model
+  compute_count                    = azurerm_oracle_autonomous_database.adbs_primary_for_crdr.compute_count
+  db_workload                      = azurerm_oracle_autonomous_database.adbs_primary_for_crdr.db_workload
+  db_version                       = azurerm_oracle_autonomous_database.adbs_primary_for_crdr.db_version
+  admin_password                   = "TestPass#2024#"
+  character_set                    = azurerm_oracle_autonomous_database.adbs_primary_for_crdr.character_set
+  national_character_set           = azurerm_oracle_autonomous_database.adbs_primary_for_crdr.national_character_set
+}
+`, a.template(data), data.RandomInteger, "eastus", "westus")
+}
+
+func (a AdbsCrossRegionDisasterRecoveryResource) requiresImport(data acceptance.TestData) string {
+	return fmt.Sprintf(`
+%s
+
+resource "azurerm_oracle_autonomous_database_cross_region_disaster_recovery" "import" {
+  name = azurerm_oracle_autonomous_database_cross_region_disaster_recovery.adbs_secondary_crdr.name
+
+  display_name                        = azurerm_oracle_autonomous_database_cross_region_disaster_recovery.adbs_secondary_crdr.name
+  location                            = azurerm_oracle_autonomous_database_cross_region_disaster_recovery.adbs_secondary_crdr.location
+  database_type                       = azurerm_oracle_autonomous_database_cross_region_disaster_recovery.adbs_secondary_crdr.database_type
+  source                              = azurerm_oracle_autonomous_database_cross_region_disaster_recovery.adbs_secondary_crdr.source
+  source_id                           = azurerm_oracle_autonomous_database_cross_region_disaster_recovery.adbs_secondary_crdr.source_id
+  source_ocid                         = azurerm_oracle_autonomous_database_cross_region_disaster_recovery.adbs_secondary_crdr.source_ocid
+  remote_disaster_recovery_type       = azurerm_oracle_autonomous_database_cross_region_disaster_recovery.adbs_secondary_crdr.remote_disaster_recovery_type
+  replicate_automatic_backups_enabled = azurerm_oracle_autonomous_database_cross_region_disaster_recovery.adbs_secondary_crdr.replicate_automatic_backups
```

Comment:
I think it shoud be:

replicate_automatic_backups_enabled = azurerm_oracle_autonomous_database_cross_region_disaster_recovery.adbs_secondary_crdr.replicate_automatic_backups_enabled